app/src/test/java/com/appunite/loudius/ui/pullrequests/PullRequestsViewModelTest.kt

```diff
@@ -75,25 +87,33 @@ class PullRequestsViewModelTest {
         clearMocks(pullRequestRepository)
         viewModel.onAction(PulLRequestsAction.RetryClick)
 
-        assertEquals(listOf(Defaults.pullRequest()), viewModel.state.pullRequests)
-        assertFalse(viewModel.state.isLoading)
+        expectThat(viewModel.state) {
+            get(PullRequestState::isLoading).isFalse()
+            get(PullRequestState::isError).isFalse()
+            get(PullRequestState::pullRequests).single().isEqualTo(Defaults.pullRequest())
+        }
     }
 
     @Test
     fun `GIVEN item id WHEN item click THEN navigate the user to reviewers`() = runTest {
         val viewModel = createViewModel()
-        assertNull(viewModel.state.navigateToReviewers)
+        expectThat(viewModel.state)
+            .get(PullRequestState::navigateToReviewers)
+            .isNull()
```

Comment:
Those are similar, both have theris downsides and upsideds
```kotlin
expectThat(viewModel.state)
            .get(PullRequestState::navigateToReviewers)
            .isNull()
```
is longer and more verbose.
```kotlin
expectThat(viewModel.state.navigateToReviewers).isNull()
```
Is shorter, the bug doesn't print the whole state in case of failure, which might make debugging a little bit harder.

Reply:
Ahh okay, yeah whole state in case of a failure feels good 👌